### DISCUSSION TITLE
I have been unable to install on a Raspberry Pi B+ (armv6l).

### DISCUSSION THREAD

**Discussion by neirfeno** [PyO3/maturin#1351] [Q&A]
I am trying to install home-assistant on a Raspberry Pi B+ (armv6l) device. One of it's dependencies is maturin. It is running the build for many hours, until it eventually exhausts all disk space (23GB free). Is this normal?

I see that there are published `sig` and `crt` assets in the release binaries, but the `gz` files are missing for `armv6l` (and some others) so I am unable to use pre-compiled binaries.

Any help would be appreciated.

I tried searching `armv6l` and `no space left`, but did not find any similar questions.

**Comment by messense** [comment]
What version of maturin did it try to install? We do have `linux_armv6l` wheels in PyPI in v0.14.5: https://pypi.org/project/maturin/0.14.5/#files
